packages/kolibri/components/pages/AppBarPage/index.vue

```diff
@@ -222,4 +246,20 @@
     background-color: white;
   }
 
+  .swipe-zone {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 5;
+    width: 30px;
+    background: red;
+    opacity: 0;
+  }
+
+  .rtl-swipe-zone {
```

Comment:
I don't think this is necessary - our RTLCSS processing creates a separate CSS asset for RTL rendering which flips the `left` attribute above to `right`.

You may have needed this during testing if you were using devserver-hot - but if you test with CSS assets actually being built, I think you'll find this isn't necessary.